app/javascript/mastodon/components/media_gallery.jsx

```diff
@@ -156,9 +157,9 @@ class Item extends PureComponent {
       const duration = attachment.getIn(['meta', 'original', 'duration']);
 
       if (attachment.get('type') === 'gifv') {
-        badges.push(<span key='gif' className='media-gallery__gifv__label'>GIF</span>);
+        badges.push(<span key='gif' className='media-gallery__alt__label'>GIF</span>);
       } else {
-        badges.push(<span key='video' className='media-gallery__gifv__label'>{formatTime(Math.floor(duration))}</span>);
+        badges.push(<span key='video' className='media-gallery__alt__label'>{formatTime(Math.floor(duration))}</span>);
```

Comment:
I understand not wanting two classes for a similar purpose, but iirc `media-gallery__alt__label` was purposefully added so that custom themes could tell the difference.

Reply:
I wasn't aware or forgot that, but the CSS duplication is unnecessary, so probably best to just add a second class to the GIFV ones. However, I'm not entirely sure, should there be a special new class for the video ones then? I don't quite understand what the use case would be.

Reply:
I added `media-gallery__alt__label--non-interactive` to the GIF and video badges, do you think that's enough for people to differentiate them or should I make the classes more specific?

Reply:
I think it is enough, although the name does seem a bit misleading.